extensions/who-is-visiting-this-content/manifest.json

```diff
@@ -0,0 +1,3107 @@
+        "Date/Publication": "2019-04-21 12:20:03 UTC",
+        "Built": "R 4.3.0; ; 2023-04-11 04:31:25 UTC; unix",
+        "RemoteType": "standard",
+        "RemotePkgRef": "xtable",
+        "RemoteRef": "xtable",
+        "RemoteRepos": "https://cran.rstudio.com",
+        "RemoteReposName": "CRAN",
+        "RemotePkgPlatform": "aarch64-apple-darwin20",
+        "RemoteSha": "1.8-4"
+      }
+    },
+    "yaml": {
+      "Source": "CRAN",
+      "Repository": "https://cran.rstudio.com",
+      "description": {
+        "Package": "yaml",
+        "Type": "Package",
+        "Title": "Methods to Convert R Data to YAML and Back",
+        "Date": "2024-07-22",
+        "Version": "2.3.10",
+        "Suggests": "RUnit",
+        "Author": "Shawn P Garbett [aut], Jeremy Stephens [aut, cre], Kirill Simonov [aut], Yihui Xie [ctb],\n  Zhuoer Dong [ctb], Hadley Wickham [ctb], Jeffrey Horner [ctb], reikoch [ctb],\n  Will Beasley [ctb], Brendan O'Connor [ctb], Gregory R. Warnes [ctb],\n  Michael Quinn [ctb], Zhian N. Kamvar [ctb], Charlie Gao [ctb]",
+        "Maintainer": "Shawn Garbett <shawn.garbett@vumc.org>",
+        "License": "BSD_3_clause + file LICENSE",
+        "Description": "Implements the 'libyaml' 'YAML' 1.1 parser and emitter\n  (<https://pyyaml.org/wiki/LibYAML>) for R.",
+        "URL": "https://github.com/vubiostat/r-yaml/",
+        "BugReports": "https://github.com/vubiostat/r-yaml/issues",
+        "NeedsCompilation": "yes",
+        "Packaged": "2024-07-22 15:44:18 UTC; garbetsp",
+        "Repository": "CRAN",
+        "Date/Publication": "2024-07-26 15:10:02 UTC",
+        "Built": "R 4.3.3; aarch64-apple-darwin20; 2025-01-25 21:39:45 UTC; unix",
+        "Archs": "yaml.so.dSYM",
+        "RemoteType": "standard",
+        "RemotePkgRef": "yaml",
+        "RemoteRef": "yaml",
+        "RemoteRepos": "https://cran.rstudio.com",
+        "RemoteReposName": "CRAN",
+        "RemotePkgPlatform": "aarch64-apple-darwin20",
+        "RemoteSha": "2.3.10"
+      }
+    }
+  },
+  "files": {
+    "app.R": {
+      "checksum": "9503ad4badfebc475f44466b7d7c8ef7"
+    },
+    "get_usage.R": {
+      "checksum": "5b2476731ce8e473c09abd1c593ccfa7"
+    }
+  },
+  "users": null,
+  "extension": {
+    "name": "content-usage-metrics-dashboard",
+    "title": "Content Usage Metrics Dashboard",
+    "description": "See what content is popular on your Connect instance. Dive into details for a content item to learn who's viewing it, and when.",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/who-is-visiting-this-content",
+    "tags": ["R", "shiny", "usage-metrics"],
```

Comment:
We don't have any tags yet, so this is going to fail linting if / when this is added to the workflow (just a heads up)